javax-servlets/src/test/java/com/baeldung.servlets/LiveTest.java

```diff
@@ -13,7 +13,7 @@
 
 import static org.junit.Assert.assertEquals;
 
-public class FormServletTest {
+public class LiveTest {
```

Comment:
I meant that it should stick to convention *LiveTest so let's name it FormServletLiveTest